src/fortran/client/script_interfaces.inc

```diff
@@ -73,6 +90,24 @@ interface
   end function set_script_c
 end interface
 
+interface
+  function set_script_multigpu_c( c_client, key, key_length, script, script_length, & 
+                                  first_gpu, num_gpus) bind(c, name="set_script_multigpu")
+    use iso_c_binding, only : c_ptr, c_size_t, c_char, c_int
+    import :: enum_kind
+    integer(kind=enum_kind)                   :: set_script_multigpu_c
+    type(c_ptr), value,            intent(in) :: c_client          !< Initialized SmartRedis client
+    character(kind=c_char),        intent(in) :: key(*)            !< The key to use to place the script
+    integer(kind=c_size_t), value, intent(in) :: key_length        !< The length of the key c-string, excluding null
+    character(kind=c_char),        intent(in) :: script(*)     !< The file storing the script
```

Comment:
Doc at the end of this line and the next don't line up properly. (Check `set_script_c` to see if this is a copy/paste of problematic source)

Reply:
Fixed.